releasenotes/notes/circuit-compose-c320c4057cd2bb8f.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    A new ``compose`` method is added to ``QuantumCircuit``. It allows
+    composition of two quantum circuits without having to turn one into
+    a gate or instruction. It also allows permutations of qubits/clbits
+    at the point of composition, as well as optional inplace modification.
+deprecations:
+  - |
+    The ``DAGCircuit.compose()`` method now takes a list of qubits/clbits
+    that specify the positional order of bits to compose onto. The
+    dictionary-based method of mapping using ``edge_map`` arg is
+    deprecated.
```

Comment:
These look to be duplicated from #4762 .

Reply:
Removed in 5db52f1cc